### PR TITLE
feat(a lot): go read the commits

### PR DESCRIPTION
**Pull request by TheGamer1002**

### CODE REVIEW

File: src/main/java/frc/robot/FieldConstants.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+ * <br>
+ * Length refers to the <i>x</i> direction (as described by wpilib) <br>
+ * Width refers to the <i>y</i> direction (as described by wpilib)
+ */
+public class FieldConstants {
+  public static double fieldLength = Units.inchesToMeters(651.223);
+  public static double fieldWidth = Units.inchesToMeters(323.277);
+  public static double wingX = Units.inchesToMeters(229.201);
+  public static double podiumX = Units.inchesToMeters(126.75);
+  public static double startingLineX = Units.inchesToMeters(74.111);
```
Comment (NyxAlexandra) [inline src/main/java/frc/robot/FieldConstants.java:27]: why not `final`?

**Review by NyxAlexandra — CHANGES_REQUESTED** [review]
see comment

File: src/main/java/frc/robot/FieldConstants.java (hunk 1 above)
Comment (NyxAlexandra) [reply src/main/java/frc/robot/FieldConstants.java:27]: additionally, should be IN SCREAMING_SNAKE_CASE

Comment (TheGamer1002) [reply src/main/java/frc/robot/FieldConstants.java:27]: I copied and pasted from somewhere else. I can add final modifiers too.

**Review by NyxAlexandra — APPROVED** [review]
lgtm

**Review by NyxAlexandra — APPROVED** [review]
test 123

**Review by NyxAlexandra — APPROVED** [review]
lgtm part 2

**Comment by TheGamer1002** [comment]
What does igtm mean

**Pull request merged**